net/ieee8021x/files/ieee8021x.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/sh
+
+[ -n "$INCLUDE_ONLY" ] || {
+	. /lib/functions.sh
+	. /lib/functions/network.sh
+	. ../netifd-proto.sh
+	init_proto "$@"
+}
+
+remove_empty_line() {
+	echo "$1" | sed '/^[[:space:]]*$/d'
+}
+
+ieee8021x_exitcode_tostring() {
+	local errorcode=$1
+	[ -n "$errorcode" ] || errorcode=5
+
+	case "$errorcode" in
+		0) echo "OK" ;;
+		1) echo "FATAL_ERROR" ;;
+		5) echo "USER_REQUEST" ;;
+		*) echo "UNKNOWN_ERROR" ;;
+	esac
+}
+
+_hostapd_common() {
+	local ifname="$1"
+
+	_config="/var/run/hostapd-$ifname.conf"
+	_pid="/var/run/wired-$ifname.pid"
```

Comment:
What do we do if the ifname is not there. And therefore no configuration can be written.